js/cronsync.js

```diff
@@ -250,11 +252,11 @@ CronSync.prototype = {
     // sync if it is sufficiently recent.
 
     // - Initiate a sync of the folder covering the desired time range.
-    this._LOG.syncAccount_begin(account.id);
-    this._LOG.syncAccountHeaders_begin(account.id, null);
+    logic(scope, 'syncAccount_begin');
```

Comment:
I do like colons better than underscores for that purpose, and `slog` was originally colon-happy for the same reason. For this specific use case ("begin" and "end"), I designed Logic's promise helpers (`async` and `await`) with the intent that they replace non-semantic 'begin and 'end' events like this. That allows us to track which async events are going on at any given time, and automatically trap errors associated with specific async tasks.